ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala

```diff
@@ -381,7 +381,7 @@ final class SandboxServer(
           maxDeduplicationDurationEnforced = false,
         ),
         contractIdFeatures = ExperimentalContractIds.of(
-          v0 = ExperimentalContractIds.ContractIdV0Support.SUPPORTED,
+          v0 = ExperimentalContractIds.ContractIdV0Support.NOT_SUPPORTED,
```

Comment:
I’ll drop the feature flag in a follow-up. Just trying to not increase the diff here further than necessary.